libbable.js

```diff
@@ -0,0 +1,106 @@
+class Libbable {
+
+  constructor( documentRef ) {
+    this.document = documentRef
+
+    this.libs = this.getElementsByClassName( "fillable" )
+    this.repeats = this.getElementsByClassName( "repeatFill" )
+
+    this.fields = this.createFields()
+  }
+
+
+  getElementsByClassName( className ) {
```

Comment:
the thing returned by this method is "Array-Like", but it's not an `Array`. Doing this to the result allows us to treat the result as a true JavaScript `Array<HTMLElement>`, on which we can perform higher-order methods like `.filter`, `map`, etc